src/multiinstances/internal/ipc/ipcchannel.cpp

```diff
@@ -57,9 +61,83 @@ bool IpcChannel::send(const Msg& msg)
     return m_selfSocket->send(msg);
 }
 
+void IpcChannel::response(const QString& method, const QStringList& args, const ID& destID)
+{
+    Msg res;
+    res.destID = destID;
+    res.type = MsgType::Response;
+    res.method = method;
+    res.args = args;
+    send(res);
+}
+
+void IpcChannel::broadcast(const QString& method, const QStringList& args)
+{
+    Msg res;
+    res.destID = BROADCAST_ID;
+    res.type = MsgType::Notify;
+    res.method = method;
+    res.args = args;
+    send(res);
+}
+
+Code IpcChannel::syncRequestToAll(const QString& method, const QStringList& args, const OnReceived& onReceived)
```

Comment:
might make sense to call it syncBroadcast

I'm a bit curious to know what is the use case of this method? Kinda to broadcast message and make sure that everyone has received it?

Reply:
Just broadcast not suitable for this case, because the term broadcast often using just for notify. In this case, we asking all (do request for all) and waiting for an answer (response) from all.

Like, has anyone opened this one score?
not
not
yes
not